app/controllers/request_controller.rb

```diff
@@ -924,7 +924,10 @@ def render_new_compose(batch)
     parameters = ActionController::Parameters.new(message_params)
     parameters.permit(:body, :default_letter)
 
-    @outgoing_message = @info_request.outgoing_messages.build(message_params)
+    @outgoing_message =
+      @info_request.
+        outgoing_messages.
+          build(message_params.merge(:info_request => @info_request))
```

Comment:
Ah, so this isn't exactly the right solution. It should be `parameters.merge(:info_request => @info_request)` to make use of the `ActionController::Parameters#permit` filtering.

Doing this causes `ActiveModel::ForbiddenAttributes` errors:

```
     Failure/Error: @info_request.

     ActiveModel::ForbiddenAttributes:
       ActiveModel::ForbiddenAttributes
     # /home/vagrant/bundle/ruby/1.9.1/gems/strong_parameters-0.2.3/lib/active_model/forbidden_attributes_protection.rb:11:in `sanitize_for_mass_assignment'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/attribute_assignment.rb:75:in `assign_attributes'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/base.rb:498:in `initialize'
     # /home/vagrant/bundle/ruby/1.9.1/gems/actionpack-3.2.22.2/lib/action_dispatch/routing/url_for.rb:103:in `initialize'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/reflection.rb:183:in `new'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/reflection.rb:183:in `build_association'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/associations/association.rb:239:in `build_record'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/associations/collection_association.rb:112:in `build'
     # /home/vagrant/bundle/ruby/1.9.1/gems/activerecord-3.2.22.2/lib/active_record/associations/collection_proxy.rb:46:in `build'
     # ./app/controllers/request_controller.rb:928:in `render_new_compose'
     # ./app/controllers/request_controller.rb:310:in `new'
```